lib/mac/CMakeLists.txt

```diff
@@ -53,44 +57,55 @@ if (ACACIA_PYTHON OR ACACIA_NODEJS)
   include(UseSWIG)
 endif()
 
-# SWIG Instructions to build a python module called "acacia_axapi"
+# Build and install a python module called "acacia_axapi" using setuptools
 
 if (ACACIA_PYTHON)
-  find_package(Python3 COMPONENTS Interpreter Development)
-
-  include_directories(${CMAKE_CURRENT_SOURCE_DIR})
-  include_directories(${Python3_INCLUDE_DIRS})
+  configure_file(python/setup.py.in python/setup.py)
+  configure_file(python/pyproject.toml python/pyproject.toml)
+  configure_file(acacia_axapi.i python/acacia_axapi/acacia_axapi.i)
```

Comment:
this two files, `pyproject.toml` and `acacia_axapi.i` aren't actually configured, I'm curious why you have these two lines here?

Reply:
It's just a lazy way to copy those files into the build directory, plus I figured we might later have things to configure them with (e.g. the version number).

Reply:
ok that's fine :)